service/patch.go

```diff
@@ -86,8 +86,13 @@ func (uis *UIServer) patchPage(w http.ResponseWriter, r *http.Request) {
 		if err != nil {
 			uis.LoggedError(w, r, http.StatusInternalServerError, errors.Wrap(err, "error finding commit queue"))
 		}
-		if cq != nil {
-			commitQueuePosition = cq.FindItem(projCtx.Patch.Id.Hex())
+		if cq == nil {
+			uis.LoggedError(w, r, http.StatusNotFound, errors.New("commit queue not found"))
```

Comment:
what happens when we write more than one error to the writer? (Like here, a commit queue that doesn't exist will have err != nil and cq == nil, I think)

Reply:
Sorry, this should be clearer. Multiple errors can be logged, but you're right that the same case above would hit two logically equal messages.